components/forms/Toggle/react.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import s from './style.module.scss'
+
+
+const Toggle = ({
+  checked, onChange,
+}) => (
+
+  <div
+    role="checkbox"
+    aria-checked={checked}
+    tabIndex={0}
+    onClick={onChange}
+    onKeyPress={onChange}
+    className={s.toggle}
+  >
+    <label className={s.toggle__switch} htmlFor="toggle">
+      <input className={s.toggle__input} type="checkbox" id="toggle" />
```

Comment:
IDs are unique and should not be used here

Reply:
If a `<label>` wraps an `<input>`, you do not need to use the `for` attribute, so if that's all it was for, you can safely remove it